driver-tests/pom.xml

```diff
@@ -77,13 +72,21 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
-
+            <plugin>
+                <groupId>net.nicoulaj.maven.plugins</groupId>
+                <artifactId>checksum-maven-plugin</artifactId>
+                <version>1.7</version>
+                <executions>
+                    <execution>
+                        <id>source-release-checksum</id>
+                        <phase>none</phase>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
Note: this change actually fixes the "basedir... does not exist" issues originally described in the ticket.